src/views/Home.jsx

```diff
@@ -1,11 +1,26 @@
 import './Home.css';
+import { useNavigate } from 'react-router-dom';
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+
+export function Home({ makeNewList }) {
+	const navigate = useNavigate();
+
+	// use callback function so the newToken that's generated in this child component
+	// can be passed up to the parent component (App.js)
+	function handleClick() {
+		const newToken = generateToken();
+		makeNewList(newToken);
+		navigate('/list');
+	}
 
-export function Home() {
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
+			<button type="button" onClick={handleClick}>
```

Comment:
One thing to consider here as a small change, would be to update the button to make it accessible! https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
I'm finding some conflicting advice on adding `role=button` to the button (that it's implicit for native button elements, and since in this case, the button is not a toggle). Do you recommend a different kind of change, especially since the button redirects after generating the token?